tests/phpunit/stubs/stub-wpml-translation-element-factory.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class WPML_Translation_Element_Factory {
```

Comment:
@BruceOnTheGo all these stubs cause inspection problems (like "multiple definitions of {class_name}").

In this case, I think you can give it a more specific name, to avoid this issue, especially considering that in the future we may need something similar for other element types. 

Something like `WPML_Stub_Post_Element_Factory` should work.

Reply:
Cool. Thanks!